docs/hugo/content/guide/aso-controller-settings-options.md

```diff
@@ -13,7 +13,7 @@ The supported options are:
 ### AZURE_SUBSCRIPTION_ID
 
 Azure subscription the operator will use for ARM communication if 
-no [more specific](multitenant-deployment.md#single-operator-multitenancy-default-recommended)
+no [more specific](authentication/credential-scope.md)
```

Comment:
Don't include the `.md` suffix on any links - they'll pass validation and then break when Hugo renders the site.

```suggestion
no [more specific](authentication/credential-scope)
```

Reply:
I haven't worked out (yet!) how to set up `markdown-link-check` to break on these.

Reply:
Fixed